RoomBookingSystem/routes/api.php

```diff
@@ -1,7 +1,20 @@
 <?php
 
 use App\Http\Controllers\AuthController;
+use App\Http\Controllers\UserController;
 use Illuminate\Support\Facades\Route;
 
 Route::post('/login', [AuthController::class, 'login']);
 Route::middleware('auth:api')->post('/logout', [AuthController::class, 'logout']);
+
+Route::prefix('/user')->group(function () {
+    Route::middleware('auth:api')->group(function () {
+        Route::post('/createUser', [UserController::class, 'createUser']);
+        Route::get('/getUser/{user}', [UserController::class, 'getUser']);
+        Route::get('/getAllUsers', [UserController::class, 'getAllUsers']);
+        Route::put('/editUser/{user}', [UserController::class, 'editUser']);
+        Route::put('/changePassword/{user}', [UserController::class, 'changePassword']);
+        Route::delete('/deleteUser/{user}', [UserController::class, 'deleteUser']);
+
+    });
+});
```

Comment:
I would recommend switching the middleware and prefix groups, because then you don't need to write the middleware group every time you make a new prefix, instead you add the prefix inside the middleware group.
```suggestion
Route::middleware('auth:api')->group(function () {
    Route::prefix('/user')->group(function () {
        Route::post('/createUser', [UserController::class, 'createUser']);
        Route::get('/getUser/{user}', [UserController::class, 'getUser']);
        Route::get('/getAllUsers', [UserController::class, 'getAllUsers']);
        Route::put('/editUser/{user}', [UserController::class, 'editUser']);
        Route::put('/changePassword/{user}', [UserController::class, 'changePassword']);
        Route::delete('/deleteUser/{user}', [UserController::class, 'deleteUser']);

    });
});
```